packages/core/src/Shimmer/constants.ts

```diff
@@ -0,0 +1,26 @@
+export enum VariationType {
+  Origin = 'origin',
```

Comment:
nb: I prefer the name original or base to be consist with the rest of the DS colour naming

Reply:
I changed it to `Base`. Thanks